src/index.html

```diff
@@ -1,17 +1,113 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
-</html>
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" href="style.css">
+   <title>HTML Form</title>
+</head>
+<body>
+   <script type="text/javascript" src="./main.js"></script>
+   <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST" onsubmit="onSubmit()">
+
+      <fieldset class="block">
+         <legend>Personal information</legend>
+         <label class="form-field" for="" id="">
+            Surname:
+            <input type="text" name="surname" required>
+         </label>
+                  
+         <label class="form-field" for="" id="">
+            Name:
+            <input type="text" name="name" required>
+         </label>
+
+         <label class="form-field" for="" id="">
+            How old are You?
+            <input type="number" name="age"
+            min="12" max="100" value="12">
+         </label>
+
+         <label class="form-field" for="" id="">
+            Full date of birth:
+            <input type="date" name="birth">
+         </label>
+
+         <label class="form-field" for="" id="">
+            I accept the term of the agreement
+            <input type="checkbox" name="term" required>
+         </label>
+      </fieldset>
+
+      <fieldset class="block">
+         <legend>Registration</legend>
+         <label class="form-field" for="" id="">
+            E-mail:
+            <input type="email" name="email" placeholder="email@example.com" required>
```

Comment:
якщо у тебе три або більше атрибута, потрібно переносити кожен атрибут на новий рядок
```suggestion
            <input 
              type="email" 
              name="email" 
              placeholder="email@example.com" 
              required
            >
```

Reply:
і так всюди